### DISCUSSION TITLE
@types/gulp-connect-php

### DISCUSSION THREAD

**Discussion by Dulux-Oz** [DefinitelyTyped/DefinitelyTyped#59385] [Request a new @types package]
Hi All,

Could someone be so kind with some advice, please, by answering one (or more) of the following questions:

1. Does anyone have the TypeScript def file for gulp-connect-php, and may I please have a copy?
2. Could someone please write the TypeScript def file for gulp-connect-php?
3. Could someone help me to write the TypeScript def file for gulp-connect-php?
4. Could someone point me towards how to use gulp-connect-php "as it is" with TypeScript?

(FTR: I looked on NPM but couldn't find anything "newer" or "more appropriate".)

Thanks in advance,

Dulux-Oz

**Comment by MysteryBlokHed** [comment]
I just opened #59392

**Reply by Dulux-Oz** [reply]
Looks like its up (as at 0100Z) - thanks MBH :-)
